Build fails on Arch Linux 64-bit

I don't know if I need to use dmd head or not, I'm using the default dmd 2.066. Output from make:

    dmd -c -ofobj/sdc.o src/sdc/*.d src/util/*.d -m64 -w -debug -gc -unittest -Ilibd-llvm/import -Ilibd-llvm/libd/src -Ilibd-llvm/src
    dmd -c -ofobj/libd.o libd-llvm/libd/src/d/lexer.d libd-llvm/libd/src/d/exception.d libd-llvm/libd/src/d/context.d libd-llvm/libd/src/d/object.d libd-llvm/libd/src/d/location.d libd-llvm/libd/src/d/base/qualifier.d libd-llvm/libd/src/d/base/type.d libd-llvm/libd/src/d/base/node.d libd-llvm/libd/src/d/base/builtintype.d libd-llvm/libd/src/util/fastcast.d libd-llvm/libd/src/util/visitor.d libd-llvm/libd/src/util/condition.d libd-llvm/libd/src/util/utf8.d libd-llvm/libd/src/d/ast/expression.d libd-llvm/libd/src/d/ast/statement.d libd-llvm/libd/src/d/ast/type.d libd-llvm/libd/src/d/ast/declaration.d libd-llvm/libd/src/d/ast/dmodule.d libd-llvm/libd/src/d/ast/identifier.d libd-llvm/libd/src/d/ast/conditional.d libd-llvm/libd/src/d/ir/expression.d libd-llvm/libd/src/d/ir/symbol.d libd-llvm/libd/src/d/ir/statement.d libd-llvm/libd/src/d/ir/dscope.d libd-llvm/libd/src/d/ir/type.d libd-llvm/libd/src/d/parser/expression.d libd-llvm/libd/src/d/parser/base.d libd-llvm/libd/src/d/parser/statement.d libd-llvm/libd/src/d/parser/ambiguous.d libd-llvm/libd/src/d/parser/dtemplate.d libd-llvm/libd/src/d/parser/type.d libd-llvm/libd/src/d/parser/declaration.d libd-llvm/libd/src/d/parser/dmodule.d libd-llvm/libd/src/d/parser/adt.d libd-llvm/libd/src/d/parser/identifier.d libd-llvm/libd/src/d/parser/util.d libd-llvm/libd/src/d/parser/conditional.d libd-llvm/libd/src/d/semantic/expression.d libd-llvm/libd/src/d/semantic/symbol.d libd-llvm/libd/src/d/semantic/statement.d libd-llvm/libd/src/d/semantic/defaultinitializer.d libd-llvm/libd/src/d/semantic/evaluator.d libd-llvm/libd/src/d/semantic/dtemplate.d libd-llvm/libd/src/d/semantic/caster.d libd-llvm/libd/src/d/semantic/typepromotion.d libd-llvm/libd/src/d/semantic/sizeof.d libd-llvm/libd/src/d/semantic/scheduler.d libd-llvm/libd/src/d/semantic/type.d libd-llvm/libd/src/d/semantic/declaration.d libd-llvm/libd/src/d/semantic/dmodule.d libd-llvm/libd/src/d/semantic/mangler.d libd-llvm/libd/src/d/semantic/closure.d libd-llvm/libd/src/d/semantic/identifier.d libd-llvm/libd/src/d/semantic/semantic.d libd-llvm/libd/src/d/semantic/aliasthis.d libd-llvm/libd/src/d/semantic/vrp.d -m64 -w -debug -gc -unittest -Ilibd-llvm/libd/src
    ar rcs lib/libd.a obj/libd.o
    dmd -c -ofobj/libd-llvm.o libd-llvm/src/d/llvm/*.d libd-llvm/import/llvm/c/target.d -m64 -w -debug -gc -unittest -Ilibd-llvm/import -Ilibd-llvm/libd/src
    ar rcs lib/libd-llvm.a obj/libd-llvm.o
    gcc -o bin/sdc obj/sdc.o -m64  -lphobos2 -Llib -ld-llvm -ld `llvm-config --ldflags` `llvm-config --libs` -lstdc++ -export-dynamic -ldl -lffi -lpthread -lm -lncurses
    bin/sdc -c -o obj/rt/eh.o libsdrt/src/d/rt/eh.d  -Ilibsdrt/src
    util.json.JSONParseException@src/util/json.d(187): Expected " at line 1 column 3!
    ----------------
    bin/sdc(util.json.JSON[immutable(char)[]] util.json.JSONReader!(immutable(char)[]).JSONReader.parseObject()+0x8a) [0xc59c3a]
    bin/sdc(util.json.JSON util.json.JSONReader!(immutable(char)[]).JSONReader.parseValue()+0x220) [0xc59f80]
    bin/sdc(util.json.JSON util.json.JSONReader!(immutable(char)[]).JSONReader.parseJSON()+0x62) [0xc5a0d2]
    bin/sdc(util.json.JSON util.json.parseJSON!(immutable(char)[]).parseJSON(immutable(char)[])+0x58) [0xc59130]
    bin/sdc(util.json.JSON sdc.conf.getConf(immutable(char)[])+0x85) [0xc3849d]
    bin/sdc(util.json.JSON sdc.conf.buildConf()+0x1ca) [0xc383a2]
    bin/sdc(_Dmain+0x28) [0xc38ca8]
    /usr/lib/libphobos2.so.0.66(_D2rt6dmain211_d_run_mainUiPPaPUAAaZiZ6runAllMFZ9__lambda1MFZv+0x28) [0x7fdcbf4145e0]
    /usr/lib/libphobos2.so.0.66(void rt.dmain2._d_run_main(int, char**, extern (C) int function(char[][])*).tryExec(scope void delegate())+0x2d) [0x7fdcbf414525]
    /usr/lib/libphobos2.so.0.66(void rt.dmain2._d_run_main(int, char**, extern (C) int function(char[][])*).runAll()+0x2d) [0x7fdcbf414585]
    /usr/lib/libphobos2.so.0.66(void rt.dmain2._d_run_main(int, char**, extern (C) int function(char[][])*).tryExec(scope void delegate())+0x2d) [0x7fdcbf414525]
    /usr/lib/libphobos2.so.0.66(_d_run_main+0x192) [0x7fdcbf41449a]
    bin/sdc(main+0x25) [0xc3ed8d]
    /usr/lib/libc.so.6(__libc_start_main+0xf0) [0x7fdcbdc51040]
    libsdrt/makefile.common:16: recipe for target 'obj/rt/eh.o' failed
    make: *** [obj/rt/eh.o] Error 1

--- a/makefile
+++ b/makefile
@@ -47,7 +47,7 @@ obj/sdc.o: $(SOURCE)
 
 bin/sdc.conf:
 	@mkdir -p bin
-	printf "{\n\t\"includePath\": [\"$(PWD)/libs\", \".\"],\n\t\"libPath\": [\"$(PWD)/lib\"],\n}" > $@
+	printf "{\n\t\"includePath\": [\"$(PWD)/libs\", \".\"],\n\t\"libPath\": [\"$(PWD)/lib\"],\n}\n" > $@
 
 clean:
 	rm -rf obj lib $(SDC)

--- a/makefile
+++ b/makefile
@@ -47,7 +47,7 @@ obj/sdc.o: $(SOURCE)
 
 bin/sdc.conf:
 	@mkdir -p bin
-	echo "{\n\t\"includePath\": [\"$(PWD)/libs\", \".\"],\n\t\"libPath\": [\"$(PWD)/lib\"],\n}" > $@
+	printf "{\n\t\"includePath\": [\"$(PWD)/libs\", \".\"],\n\t\"libPath\": [\"$(PWD)/lib\"],\n}" > $@
 
 clean:
 	rm -rf obj lib $(SDC)

